hs/ExportResponse.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module ExportResponse where
+import           Text.Blaze ((!))
+import qualified Text.Blaze.Html5 as H
+import qualified Text.Blaze.Html5.Attributes as A
+import Happstack.Server
+import MakeElements
+import MasterTemplate
+import Graphics.PDF
+
+exportResponse :: ServerPart Response
+exportResponse =
+   ok $ toResponse $
+    masterTemplate "Courseography - SVG serving test!"
+                [H.meta ! A.name "keywords"
+                        ! A.content "",
+                 aboutLinks
+                ]
+                (do
+                    header "export"
+		    exportHtml
```

Comment:
We haven't exactly set any constraints on `do` block style, and quite frankly I think I've made my own up somehow (which is probably a bad thing), but please align these two lines.